spec/model/user_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @user = User.new(name: 'Tom', email: 'test@gmail.com', password: '1234')
+  end
+
+  it 'Should be valid' do
+    @user.save
+    expect(@user).to be_valid
+  end
```

Comment:
[OPTIONAL] - Maybe you can describe better what it means for the user to be valid (although it is pretty self-explanatory from the test. But it's good practice.

Reply:
yeah sure. :+1: :pray: